manipulation/models/marks-table/table.mtl

```diff
@@ -0,0 +1,13 @@
+# Blender MTL File: 'edit_shelf_geom.blend'
```

Comment:
do the table models in drake really not work?  i really dislike adding all these asset files; they'll be in git history forever.

Reply:
Switched to using only models from Drake, except for the fixed PR2 urdf.